live-usb-creator/README.md

```diff
@@ -9,15 +9,17 @@ Three dependencies need to be fetched out-of-band.
 
 Set the following files in place in the same directory as the `Vagrantfile`.
 
-* CodeSafe-linux64-dev-12.50.2.iso (2.6GB): supplied by the HSM vendor.
+* SecWorld_Lin64-12.60.11.iso (316M): supplied by the HSM vendor
```

Comment:
What's the difference between these two ISO images, @oreparaz ?

Reply:
nCipher changed the way they package the different components in 12.60. Codesafe is now a standalone .iso